protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt

```diff
@@ -0,0 +1,8 @@
+#Wed Dec 17 17:11:53 CET 2014
+#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd
```

Comment:
Just stumbled across this file, I had forgotten about it before. Is this really required (with bla bla bla) or can it be removed?